blog/2024-04-17-10101-vision-and-status.md

```diff
@@ -0,0 +1,167 @@
+
+**Our Team's Track Record**
+
+Our team is not new to the world of cryptographic protocols and Bitcoin. Starting as an R&D lab in 2018, we initially focused on trustless cross-chain trading, achieving milestones such as the world’s first ERC-20 and Bitcoin atomic swap. Our expertise expanded with projects that bridged Monero and Bitcoin and experiments in trustless lending on the Liquid network. Learning from these experiences, we launched ItchySats, the precursor to what 10101 is today.
+
+**Current Availability**
+
+10101 is live and operational. You can start trading today by downloading our app for Android and iOS. Visit [10101 Finance](https://10101.finance/) for direct links to your favorite app store and get started in no time.
+
+### Where We're Headed: Expanding Horizons at 10101
+
+### Short-term Goals: Perfecting the Platform
+
+Our journey with the public beta of 10101 on Android and iOS has been encouraging, but we recognize there's more to be done to refine our service. We are committed to reinstating key features and introducing new functionalities:
+
+- **Reintegration of Lightning Network:** Earlier this year, we had to remove support for the Lightning Network. We plan to reintroduce this capability, enabling faster, cheaper Bitcoin transactions which are critical for efficient microtransactions and real-time trading.
+- **Re-launching USDp:** Our synthetic stable coin, USDp, will return to the platform. This feature allows users to hedge against Bitcoin’s volatility without leaving Bitcoin’s ecosystem, maintaining value in USD while benefiting from the security and potential of Bitcoin. USDp's integration with the Lightning Network further enhances its utility, making it transferable as sats.
+
+The next major milestone for us is moving out of beta. Achieving this status means reaching feature completion and ensuring platform stability, a milestone we believe is within close reach.
+
+### Long-term Vision: Untethering Self-sovereign Bitcoin Finance
+
+![Road into the future](/2024-17-04-10101-vision-and-status/longterm-vision-road-to-future.png)
+
+Looking beyond immediate improvements, our goal is to address some of the fundamental challenges in the crypto trading space:
+
+- **Innovating Liquidity Pools:** Current channel-based trading is limited to two-party interactions. We aim to expand this by developing liquidity pools directly on the Bitcoin network. Although this research is nascent, technologies like BitVM and potential softfork enhancements like OP_CTV or OP_CAT show promise. These advancements will pave the way for a more interconnected and fluid trading environment.
+- **One App, All Things Bitcoin:** In the future, 10101 aims to be the all-encompassing app for everything Bitcoin-related. From trading without counterparty risks to managing both hot and cold storage, or even making everyday purchases with Bitcoin or USDp, we envision 10101 as the ultimate tool for Bitcoin enthusiasts.
```

Comment:
I see the appeal, but this goes against the rest of the blog post. Here we are talking about a very different 10101 than the one that is described above.

It's a cool slogan, but I think we did well to move away from it because we can't lose focus at this stage.